packages/blocto-sdk/src/constants.ts

```diff
@@ -37,6 +37,7 @@ export const ETH_CHAIN_ID_CHAIN_MAPPING: Mapping = {
   // Ethereum
   1: 'ethereum',
   4: 'ethereum',
```

Comment:
Should we remove chiainId=4?

Reply:
I thought about it, remove might be better since BE no longer supports too.